src/main/groovy/wooga/gradle/paket/unity/internal/UnityDllMetaFile.groovy

```diff
@@ -0,0 +1,65 @@
+package wooga.gradle.paket.unity.internal
+
+/***
+ * A representation of a .meta file generated by Unity during an import
+ */
+class UnityDllMetaFile {
+
+    String guid
+    Boolean autoLoaded = true
+    Boolean validateReferences = false
+
+    @Override
+    String toString() {
+        return """fileFormatVersion: 2
+guid: ${guid}
+PluginImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  iconMap: {}
+  executionOrder: {}  
+  isPreloaded: 0
+  isOverridable: 0
+  isExplicitlyReferenced: ${(autoLoaded ? "0" : "1")}
+  validateReferences: ${(validateReferences ? "1" : "0")}
+  platformData:
+  - first:
+      Any:
+    second:
+      enabled: 1
+      settings: {}
+  - first:
+      Editor: Editor
+    second:
+      enabled: 0
+      settings:
+        DefaultValueInitialized: true
+  - first:
+      Windows Store Apps: WindowsStoreApps
+    second:
+      enabled: 0
+      settings:
+        CPU: AnyCPU
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
+        """.trim()
```

Comment:
Please use the `stripIndent` method. You can also start the line with `\` so you don't need the dangling first line or continue calling `.trim`. But it might be better in general to use a groovy map instead and serialize to yaml when saving.

Reply:
Sure, I will invoke `stripIndent`.